spec/models/conference_spec.rb

```diff
@@ -778,6 +778,35 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+
+    before(:each) do
+      @conference = create(:conference)
+
+      selected_schedule = create(:schedule, program: @conference.program)
+      @conference.program.update_attributes!(selected_schedule: selected_schedule) 
+
+      confirmed_unscheduled_event = create(:event, program: @conference.program, state: 'confirmed')
```

Comment:
all this variables are only needed for the first test, so do not create them for both tests, the same with the assignment of the `result` variable :wink: